test/test_polkit_allowlist.py

```diff
@@ -0,0 +1,40 @@
+import os
+import os.path
```

Comment:
maybe use of `pathlib` will be better option

Reply:
I don't personally see how it'd provide a benefit over a single call to `os.path.join`.

But I don't care much either way, so I'll use that if that's what you prefer.